skins/smoothyfinance.scss

```
@@ -0,0 +1,10 @@
+.smoothy {
```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.smoothyfinance {
```